code/controllers/subsystem/shuttle.dm

```diff
@@ -399,12 +399,14 @@ SUBSYSTEM_DEF(shuttle)
 	return 0	//dock successful
 
 
-/datum/controller/subsystem/shuttle/proc/moveShuttle(shuttleId, dockId, timed)
+/datum/controller/subsystem/shuttle/proc/moveShuttle(shuttleId, dockId, timed, no_stowaways)
 	var/obj/docking_port/mobile/M = getShuttle(shuttleId)
 	var/obj/docking_port/stationary/D = getDock(dockId)
 
 	if(!M)
-		return 1
+		return 1	//invalid shuttle
+	if (no_stowaways && !M.check_exile_pass())
+		return 2	//unable to comply
```

Comment:
What does 2 mean in this scenario. I know it's not your code but it should be defined

Reply:
Ok new commit adresses it.